chaos-monkey-spring-boot/src/main/java/de/codecentric/spring/boot/chaos/monkey/component/MetricEventPublisher.java

```diff
@@ -25,7 +25,8 @@ public void publishMetricEvent(MetricType metricType, String... tags) {
     }
 
     public void publishMetricEvent(MetricType metricType, AtomicInteger atomicTimeoutGauge) {
-        MetricEvent metricEvent = new MetricEvent(this, metricType, atomicTimeoutGauge);
+        //MetricEvent constructor has been deprecated. So falling back to the internally used Constructor
+        MetricEvent metricEvent = new MetricEvent(this, metricType, atomicTimeoutGauge!=null? atomicTimeoutGauge.longValue() : -1,null);
```

Comment:
❓ Does it make sense to extract the conditional to a separate variable for better readability?
❓ Any change you can run a formatter on the code? We tend to use IntelliJ's standard formatter (it is not yet implemented in the CI pipeline but we will change that soon, so this ain't a manual process anymore).